astropy/io/fits/diff.py

```diff
@@ -29,7 +29,7 @@
 from .card import Card, BLANK_CARD
 from .header import Header
 # HDUList is used in one of the doctests
-from .hdu.hdulist import fitsopen, HDUList  # pylint: disable=W0611
```

Comment:
@astrofrog - @embray's commit has already solved it 
(btw a somewhat related question: the doc examples don't run ever (except during a doctest testing), right? So imports like ``HDUList`` in this example, when the modules is actually not used in the code itself, is a bit more optimal to do in the doc rather than at the top of the module?)